### PR TITLE
Add voltage init ratio tap transformers modifications

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/components/dialogs/network-modifications/voltage-init-modification/voltage-init-modification-dialog.tsx [hunk 1]
```diff
@@ -164,18 +205,40 @@ const VoltageInitModificationDialog: FunctionComponent<
...
+                    />
+                );
+            } else if (currentTab === EquipmentTypeTabs.TRANSFORMER_TAB) {
+                let rowData: TransformerRowData[] = [];
+                if (editData) {
+                    editData.transformers.forEach((m: TransformerData) => {
+                        let row: TransformerRowData = {
+                            ID: m.transformerId,
+                            [RATIO_TAP_CHANGER_POSITION]:
+                                m.ratioTapChangerPosition,
```
Comment (ghazwarhili) [inline src/components/dialogs/network-modifications/voltage-init-modification/voltage-init-modification-dialog.tsx:224]: if m is null ?
```
 if (m.ratioTapChangerPosition) {
row[RATIO_TAP_CHANGER_POSITION] = m.ratioTapChangerPosition;}
 if (m.legSide) {
row[LEG_SIDE] = m.legSide; }
```

**Review by ghazwarhili — CHANGES_REQUESTED** [review]

File: src/components/dialogs/network-modifications/voltage-init-modification/voltage-init-modification-dialog.tsx (hunk 1 above)
Comment (FranckLecuyer) [reply src/components/dialogs/network-modifications/voltage-init-modification/voltage-init-modification-dialog.tsx]: Changed (and m cannot be null)

**Review by ghazwarhili — COMMENTED** [review]
code review : ok

**Review by ghazwarhili — APPROVED** [review]

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: ok
tests: 
 - I cant see any transformer in the modification (reason of the CR).
 - maybe use the AgGrid option to fill the whole place available in the Form?

**Comment by FranckLecuyer** [comment]
> code: ok tests:
> 
>     * I cant see any transformer in the modification (reason of the CR).
You should add transformers filters in : Settings --> Voltage profile initialization --> Equipment selection --> Variable transformers
Otherwise, no transformer ratio tap will be changed ...
> 
>     * maybe use the AgGrid option to fill the whole place available in the Form? (cf search for onGridReady={onGridReady})

**Comment by FranckLecuyer** [comment]
> code: ok tests:
> 
>     * I cant see any transformer in the modification (reason of the CR).
You should add transformers filters in : Settings --> Voltage profile initialization --> Equipment selection --> Variable transformers
Otherwise, no transformer ratio tap will be changed ...
> 
>     * maybe use the AgGrid option to fill the whole place available in the Form? (cf search for onGridReady={onGridReady})


> code: ok tests:
> 
>     * 1 thing to improve : When tap changer position is 0, we display an empty (or null) value in the agGrid rather than "0". Can be tested with JUMEAY762:
> 
Fixed
> 
> ```
> INFO: 2 windings transformer with id=JUMEAY762 modified :
> INFO:      Ratio tap changer position : 0 -> 0
> ```

**Review by dbraquart — APPROVED** [review]
code: ok
tests: ok  (3WT not tested)

**Pull request merged**